### PR TITLE
List the members of the batch (read OnbardingBatch contract)

### LINKED ISSUE #3: List the members of the batch (read OnbardingBatch contract)
Show the current members of your batch in the app.

- Show it on the main page or create a page (/builders) listing all the builders registered in the BatchRegistry contract 
- Each builder should link to their personal profile

### PR DESCRIPTION
**Pull request by smrm-dev**
Fixes #3

## Description
image

## Additional Information

- [x] I have read the [contributing docs](/scaffold-eth/scaffold-eth-2/blob/main/CONTRIBUTING.md) (if this is your first contribution)
- [x] This is not a duplicate of any [existing pull request](https://github.com/scaffold-eth/scaffold-eth-2/pulls)

## Related Issues

_Closes #3_

Your ENS/address: keccack.eth

### CODE REVIEW

File: packages/nextjs/app/builders/page.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    contractName: "BatchRegistry",
+    eventName: "CheckedIn",
+    fromBlock: 115767272n,
+  });
+  // create builders list from events
+  const builders: Array<any> = [];
+  checkedInEvents?.forEach(event => {
+    const builder = event.args.builder;
+    if (!builders.includes(builder)) builders.push(builder);
+  });
```
Comment (technophile-04) [inline packages/nextjs/app/builders/page.tsx:30]: We could have even used `sets` for to handle duplicate elements 🙌

```ts
const buildersSet = new Set(checkedInEvents?.map(event => event.args.builder));
const builders = Array.from(buildersSet);
```

**Comment by technophile-04** [comment]
This is great @smrm-dev ! Maybe we should also handle responsiveness on mobile? 

<details>
<summary>
overflows on phone
</summary>

Screenshot 2024-02-19 at 2 07 44 PM

</details>

**Comment by smrm-dev** [comment]
> This is great @smrm-dev ! Maybe we should also handle responsiveness on mobile?
> overflows on phone
> 
> Screenshot 2024-02-19 at 2 07 44 PM

fixed ⬇️ 
image

**Comment by carletex** [comment]
Great stuff @smrm-dev !!

**Pull request merged**